scripts/o3de/o3de/manifest.py

```diff
@@ -16,7 +16,7 @@
 from packaging.version import Version
 from collections import deque
 
-from o3de import validation, utils, repo, compatibility
+from o3de import validation, utils, repo, compatibility, cmake
```

Comment:
This is a circular import:
The cmake script imports manifest.py

Reply:
good catch